sandbox/settings.py

```diff
@@ -43,6 +42,11 @@ class Base(Configuration):
     ALLOWED_HOSTS = []
     SECRET_KEY = values.Value(None)
 
+    # Disable Samesite flag in session and csrf cookies, because ashley is meant to
+    # run in an iframe on external websites.
```

Comment:
Please add a note here explaining why this will fail with some browsers and that it will be fixed in Django 3.1 (as you did in your PR description).

Reply:
Good idea ! Done :heavy_check_mark: